nextflow_schema.json

```diff
@@ -10,7 +10,11 @@
             "type": "object",
             "fa_icon": "fas fa-terminal",
             "description": "Define where the pipeline should find input data and save output data.",
-            "required": ["input", "outdir", "step"],
+            "required": [
+                "step",
+                "input",
+                "outdir"
+            ],
```

Comment:
prettier did the one line for me.
did tools put it back for you?

Reply:
yep, I forgot to run prettier and it didn't lint on safe